tests/system/providers/google/cloud/gcs/example_gcs_sensor.py

```diff
@@ -141,10 +140,11 @@ def mode_setter(self, value):
     # [END howto_sensor_object_exists_task]
 
     # [START howto_sensor_object_exists_task_async]
-    gcs_object_exists_async = GCSObjectExistenceAsyncSensor(
+    gcs_object_exists_async = GCSObjectExistenceSensor(
```

Comment:
The same here.
Also, if we are updating examples here (removing tasks), please make sure that changes in gcs.rst file are the same - the howto tags should have specific name

Reply:
yes i am making changes in bigquery.rst file for the corresponding START and END tag markers @VladaZakharova